build-tools-internal/src/integTest/groovy/org/elasticsearch/gradle/internal/StringTemplatePluginTest.groovy

```diff
@@ -35,12 +35,12 @@ class StringTemplatePluginTest extends AbstractGradleFuncTest {
           tasks.named("stringTemplates").configure {
             template {
               it.properties = ["Type" : "Int", "type" : "int"]
-              it.inputFile = new File("${projectDir}/src/main/p/X-Box.java.st")
+              it.inputFile = new File(projectDir, "src/main/p/X-Box.java.st")
```

Comment:
Alternatively this could just be `file("src/main/p/X-Box.java.st")` rather than using the `File` constructor and having to explicitly pass `projectDir`.

Reply:
That's cleaned - done.